### ISSUE TITLE
Syntax error raise KeyError, "object doesn't have key 'category'"

### ISSUE THREAD

**Issue by eak65** [kurtmckee/feedparser#136]
I am getting a syntax error whenever I run feedparser on my azure app service. It works locally, however, throws the error below whenever I push it to azure. 

It is running Python 3.6.4 on Azure and my local is running 3.6.5

Any ideas? 

```
Error occurred while reading WSGI handler:

Traceback (most recent call last):
  File "D:\home\python364x64\wfastcgi.py", line 791, in main
    env, handler = read_wsgi_handler(response.physical_path)
  File "D:\home\python364x64\wfastcgi.py", line 633, in read_wsgi_handler
    handler = get_wsgi_handler(os.getenv("WSGI_HANDLER"))
  File "D:\home\python364x64\wfastcgi.py", line 600, in get_wsgi_handler
    handler = __import__(module_name, fromlist=[name_list[0][0]])
  File ".\app.py", line 8, in <module>
    from newspaper import Article
  File "D:\home\python364x64\lib\site-packages\newspaper\__init__.py", line 10, in <module>
    from .api import (build, build_article, fulltext, hot, languages,
  File "D:\home\python364x64\lib\site-packages\newspaper\api.py", line 12, in <module>
    import feedparser
  File "D:\home\python364x64\lib\site-packages\feedparser.py", line 316
    raise KeyError, "object doesn't have key 'category'"
                  ^
SyntaxError: invalid syntax


StdOut: 

StdErr
```

**Comment by misterunknown** [comment]
I get the same error. There is apparently no support for python 3.6.

**Comment by bvermeulen** [comment]
Feedparser support 3.6.8 but not 3.7

**Comment by miigotu** [comment]
This happens in some python3.8 installs also, pypi has 5.2.1 and on my local machine I get the correct syntax version, but on some machines gets the 5.2.1 that is on master.

**Comment by kurtmckee** [comment]
feedparser 6.0.0 was released today and only supports Python 3.6 and higher. Moving forward there should be no issues with Python 2 syntax.
